fhir_river_api/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.7-slim
+
+# Dependencies
+RUN apt-get update \
+    && apt-get -y install build-essential
```

Comment:
I added these dependencies in the Transform because of uWSGI (dep for cleaning-scripts) or something like that so I don't think it's needed here.
Also it's shouldn't be needed anymore in the Extract because I removed the cleaning from this service.
Would you mind trying to remove them?

Reply:
Sure, no prob